website/content/docs/resources/roadmap.mdx

```diff
@@ -17,31 +17,29 @@ a release, please refer to the [GitHub Issue Tracker](https://github.com/hashico
 This is a more manual approach to reading our roadmap and includes small details
 such as bug fixes, but is much more accurate to what we're working on.
 
-### Server Production Readiness Improvements
+### Custom Pipeline
```

Comment:
This reads like we don't support custom pipelines at all! I think we should at least mention that some of this functionality is availble as a feature preview, and link to those docs.

Reply:
Updated. Does the section make sense now? Too specific?